packages/calcite-components/src/components/pagination/pagination.scss

```diff
@@ -1,42 +1,88 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+
+ * @prop --calcite-pagination-background-color-active: Specifies the background color of the pagination items while active.
+ * @prop --calcite-pagination-background-color-hover: Specifies the background color of the pagination items while hovered.
```

Comment:
remove the stateful tokens here.

Reply:
These are rendered within the component - without this the user can't style on hover.